.changeset/wicked-taxis-jam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/scroll-area": minor
+---
```

Comment:
[ask]
Is there a problem with MINOR changes?

Reply:
Fixed